pkgs/tools/networking/ocproxy/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, libevent }:
+
+let
+  version = "1.50";
```

Comment:
nitpick: it's quite common to put the version in the derivation directly below the name.